packages/sync/src/modules/Callables.php

```diff
@@ -417,6 +435,31 @@ public function maybe_sync_callables() {
 
 	}
 
+	/**
+	 * Get the callables that should always be sent, e.g. on cron.
+	 *
+	 * @return array Callables that should always be sent
+	 */
+	protected function get_always_sent_callables() {
+		$callables      = $this->get_all_callables();
+		$cron_callables = array();
+		foreach ( self::ALWAYS_SEND_UPDATES_TO_THESE_OPTIONS as $key ) {
+			if ( array_key_exists( $key, $callables ) ) {
+				$cron_callables[ $key ] = $callables[ $key ];
+				continue;
+			}
+
+			// Check for the Callable name/key for the option, if different from option name.
+			if ( array_key_exists( $key, self::OPTION_NAMES_TO_CALLABLE_NAMES ) ) {
+				$key = self::OPTION_NAMES_TO_CALLABLE_NAMES[ $key ];
```

Comment:
We seem to be mutating `$key` here, which can create unexpected results when refactoring the code.